aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -57,7 +57,9 @@ See https://github.com/dotnet/AspNetCore.Docs/issues/21193
 
 * Start Visual Studio and select **Create a new project**.
 * In the **Create a new project** dialog, select **ASP.NET Core Web App (Model-View-Controller)** > **Next**.
-* In the **Configure your new project** dialog, enter `MvcMovie` for **Project name**. It's important to name the project *MvcMovie*. Capitalization needs to match each `namespace` when code is copied.
+* In the **Configure your new project** dialog:
+  * Enter `MvcMovie` for **Project name**. It's important to name the project *MvcMovie*. Capitalization needs to match each `namespace` when code is copied.
+  * Enter a directory for the project in **Location**. The project can be located anywhere.
```

Comment:
Nit: I've never actually done that (entered a directory). Since this is VS they're not likely to be keying in folder names, they'll navigate to someplace and the folder will be entered automatically in the location field. I think you could just say it can be located anywhere, but it's up to you.